manifests/gateways/istio-ingress/templates/meshexpansion.yaml

```diff
@@ -63,6 +69,44 @@ spec:
         mode: DISABLE
 ---
 
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: meshexpansion-vs-istiod
+  namespace: {{ .Release.Namespace }}
+  labels:
+    release: {{ .Release.Name }}
+spec:
+  hosts:
+  - istio-pilot.{{ .Values.global.istioNamespace }}.svc.{{ .Values.global.proxy.clusterDomain }}
+  gateways:
+  - meshexpansion-gateway
+  tcp:
+  - match:
```

Comment:
I think you can just add another `match` section to the existing VS, but I don't think its a big difference

Reply:
actually is this meant to be `istio-pilot` in hosts -> `istiod` in destination?

Reply:
correct, fixed now, thx!